src/tests/like.test.js

```diff
@@ -0,0 +1,46 @@
+/** * @jest-environment jsdom */
+import { getLikes, addLike } from '../modules/likes.js';
+import { BASE_URL } from '../modules/utils.js';
+
+describe('getLikes', () => {
+  it('should return data from the /likes endpoint', async () => {
+    // Arrange
+    const mockSuccessResponse = {};
+    const mockJsonPromise = Promise.resolve(mockSuccessResponse);
+    const mockFetchPromise = Promise.resolve({
+      json: () => mockJsonPromise,
+    });
+    jest.spyOn(global, 'fetch').mockImplementation(() => mockFetchPromise);
+
+    // Act
+    const result = await getLikes();
+
+    // Assert
+    expect(result).toEqual(mockSuccessResponse);
+  });
+});
+
+describe('addLike', () => {
+  it('should send a POST request to the /likes endpoint', async () => {
+    // Arrange
+    const mockSuccessResponse = 'Success';
+    const mockFetchPromise = Promise.resolve({
+      text: () => mockSuccessResponse,
+    });
+    jest.spyOn(global, 'fetch').mockImplementation(() => mockFetchPromise);
+
+    // Act
+    const result = await addLike('123');
+
+    // Assert
+    expect(result).toEqual(mockSuccessResponse);
+    expect(fetch).toHaveBeenCalledWith(
+      `${BASE_URL}/likes`,
+      expect.objectContaining({
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({ item_id: '123' }),
+      }),
+    );
+  });
```

Comment:
- [ ] Good job :+1: working on the testing but please make sure that you fix all falling test 
![Screenshot from 2023-01-07 00-00-32](https://user-images.githubusercontent.com/92303891/211120182-0501d6a0-2b83-4e27-a1ab-319133ed3c86.png)

Reply:
fulfilled 💯